packages/lesswrong/components/search/SearchPageTabbed.tsx

```diff
@@ -121,6 +128,11 @@ const styles = (theme: ThemeType): JssStyles => ({
       ...theme.typography.body2,
     },
   },
+  searchHelp: {
+    [theme.breakpoints.down('sm')]: {
+      display: "none",
+    },
+  },
```

Comment:
I think this should be `xs`, or maybe even just visible on all screen sizes (not sure, but seems ok on small screens?)

Reply:
The reason I set it so high is that it's a mouse-only feature. The tooltips are not a good experience on mobile.